help-text.md

```diff
@@ -10,6 +10,8 @@ Sparks' dice-rolling commands are all subcommands of `/roll`. For example, `/rol
 
 - `/roll pbta`: rolls a Powered by the Apocalypse move. Requires the stat, including any bonuses to the roll (i.e. +1 forwards/ongoings, move-related bonuses, etc.)
 
+- `/roll wild`: rolls a Wild Words roll. Requires both the number of d6s to roll and the type of roll, and optionally the number of dice to cut.
```

Comment:
thinking we should have this be "Cutting dice coming soon" or something to that effect until you get the chance to implement it.